<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkRegionGrowingViewControls</class>
 <widget class="QWidget" name="QmitkRegionGrowingViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>541</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkRegionGrowingView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblWarning">
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
     </property>
     <property name="text">
      <string>Please select an image!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkPointListWidget" name="lstPoints">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="sliderOffsetValue">
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="offsetValueSpinBox">
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnPerformImageProcessing">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="text">
      <string>Start region grower</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkPointListWidget</class>
   <extends>QListView</extends>
   <header>QmitkPointListWidget.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">QmitkDataStorageComboBox.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>sliderOffsetValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>offsetValueSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offsetValueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderOffsetValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
